setup.cfg

```diff
@@ -0,0 +1,22 @@
+[metadata]
+name = zppy-e3sm-chem-diags
+version = 1.0.0
```

Comment:
The version could also be 0.1.0, indicating that it is not a fully mature package yet.  I strongly encourage using this version numbering scheme:
https://semver.org/
If you do, the first number of 3 (the major version) is updated when there are changes that break backward compatibility with previous versions.  The second number (the minor version) is for changes that don't break backwards compatibility, so if you update from 1.0.0 to 1.1.0, we would expect that `zppy` still works as before.  This means that a given version of zppy should be able to depend on `zppy-e3sm-chem-diags >=1.0.0,<2.0.0` and know that nothing will be broken.  The third number (the micro version) is used for bug fixes only, so 1.0.1 would have a bug fix on 1.0.0.

Reply:
@xylar the names proposed here are all good. If there is no strong preference,  I think in general it would be easier to avoid using dash `-` in the package name, but to use underscore instead, so the module name and the package name are consistent.

Reply:
I agree that underscores do tend to avoid a certain level of confusion.  For some reason, the community seems to prefer dashes over underscores.